R/utils.R

```diff
@@ -536,7 +536,7 @@ deprecated_quo_is_present <- function(quo) {
     FALSE
   } else {
     quo_expr <- rlang::get_expr(quo)
-    if (identical(quo_expr, rlang::expr(deprecated())) || identical(quo_expr, rlang::expr(lifecycle::deprecated()))) { # nolint: object_usage_linter
+    if (identical(quo_expr, rlang::expr(lifecycle::deprecated())) || identical(quo_expr, rlang::expr(lifecycle::deprecated()))) { # nolint: object_usage_linter
```

Comment:
issue: this can't be `lifecycle::deprecated()`; it's matching the expression `deprecated()` itself.  See the second branch that checks for the expression `lifecycle::deprecated()`; both should stick around.  We need to importFrom lifecycle deprecated or maybe delete and archive this function.... somewhere.